docker-traceability-plugin/src/main/resources/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction/_api.jelly

```diff
@@ -25,200 +25,227 @@
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+<div id="docker-traceability">
+  <style scoped="true">
+    /* scoped only works in FF therefore we have to add a wrapper (div#docker-traceability) */
```

Comment:
@oleg-nenashev 

* Could you tell if exist another plugin that includes its own API documentation?
* This change is not only *highlighting*, it also includes semantic in the markup
* There are a lot of plugins that includes its own CSS style in jelly templates
* If Jenkins Core includes a proposal to attend this, I'll remove the local CSS styles